src/components/TodosFilter/TodosFilter.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import { TodosContext } from '../../TodosContext';
+import { Status } from '../../types';
+
+export const TodosFilter: React.FC = () => {
+  const {
+    incompletedCount,
+    areSomeCompleted,
+    removeCompleted,
+    status,
+    setStatus,
+  } = useContext(TodosContext);
+
+  return (
+    <footer className="footer" data-cy="todosFilter">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${incompletedCount} items left`}
```

Comment:
```suggestion
        {incompletedCount} items left
```

Reply:
I also like this approach, but the linter in this and other projects asks either to use my variant or to put 'items left' on a new line